{{cookiecutter.repo_name}}/.github/workflows/release.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           cache: 'pip'
       - run: |
           ./ci/linux/create_venv.sh
+          ./ci/linux/update_dependencies.sh
```

Comment:
This is quite dangerous. We will release a version with dependencies that have not been tested with nor are in the repo. What was the reason you added this? To make sure that the release has the newest dependencies?

Reply:
update_dependencies only converts the depencency list from pyproject.toml  to requirements.txt files,  which is required for install_dependencies.sh to work.  

It doesn't change the version of the depenency that's installed. that version is (or should be) specified in pyproject.toml